concent_api/common/tests/test_api_view.py

```diff
@@ -1,28 +1,30 @@
+from logging import Logger
 import json
 import mock
 
-from django.conf                    import settings
-from django.http.response           import HttpResponse
-from django.shortcuts               import reverse
-from django.test                    import override_settings
-from django.test                    import RequestFactory
-from django.test                    import TestCase
-from django.test                    import TransactionTestCase
-from django.views.decorators.http   import require_POST
-from golem_messages                 import dump
-from golem_messages                 import load
-from golem_messages                 import message
-from golem_messages.factories       import tasks
+from django.conf import settings
+from django.http.request import HttpRequest
+from django.http.response import HttpResponse
+from django.shortcuts import reverse
+from django.test import override_settings
+from django.test import RequestFactory
+from django.test import TestCase
+from django.test import TransactionTestCase
+from django.views.decorators.http import require_POST
+from golem_messages import dump
+from golem_messages import load
+from golem_messages import message
+from golem_messages.factories import tasks
 from golem_messages.factories.tasks import ComputeTaskDefFactory
 from golem_messages.utils import encode_hex
 
-from core.exceptions                import Http400
-from core.models                    import Client
-from common.constants                import ErrorCode
-from common.decorators               import require_golem_message
-from common.decorators               import handle_errors_and_responses
-from common.helpers                  import get_current_utc_timestamp
-from common.testing_helpers          import generate_ecc_key_pair
+from core.exceptions import Http400
+from core.models import Client
+from common.constants import ErrorCode
+from common.decorators import require_golem_message
+from common.decorators import handle_errors_and_responses
+from common.helpers import get_current_utc_timestamp
+from common.testing_helpers import generate_ecc_key_pair
```

Comment:
Why are you reordering imports in this pull request? I don't say it is bad, but in future separate commit would be nice :)